README.md

```diff
@@ -47,3 +47,22 @@ If you are a committer, do the following:
 4. The CI build will run and commit the generated site to the `asf-site` branch automatically within a few minutes, from where it will also be published. CI build status mails go to the commits list.
 4. Verify the updated website works as expected by browsing it.
 
+Add a new ActiveMQ Classic Release
```

Comment:
I also think this section would be better somewhere other than the main readme. Either a new file, or the 5.x release guide page at src/release-guide.md

The latter is out of date generally, but seems the appropriate place for this should be. In either case the readme could then link to it.